csuntechlab/iSTART
# STR-236 -- FE: Update Sam to quiz + Revision Notes

# Description
Convert Samuels story into a quiz; Fix all *NEW* client revision notes.
  
# Testing
1. Go into the *Marijuana* module *index 7* and check to ensure that sam's story is now a quiz.
2. Check to ensure the quiz meets the clients request.
3. Ensure all new client revision notes are met.
